libs/geography/driver/testing/in-memory/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "@daffodil/geography/driver/testing/in-memory"
```

Comment:
I think in-memory is not a part of the "testing" package? I think it's worthy of its own driver unrelated to "testing".

Reply:
yeah that makes sense. changed